DP1.00044.001/Standard.calculations.txt

```diff
@@ -0,0 +1,3 @@
+Data are populated with 0 for time periods with no precipitation. Time periods when xxxPrecipBulk is blank or NA indicate either the sensor was not functioning or the data were redacted by data quality algorithms.
```

Comment:
In this product it's just precipBulk, right? Not xxxPrecipBulk

Reply:
Ah yes, that was just the original text from DP1.00006.001